sdcm/nemesis.py

```diff
@@ -1094,7 +1098,9 @@ def toggle_table_ics(self):
         if new_compaction_strategy in [CompactionStrategy.INCREMENTAL, CompactionStrategy.SIZE_TIERED]:
             for param in list_additional_params:
                 new_compaction_strategy_as_dict.update(param)
-        cmd = "ALTER TABLE {keyspace_table} WITH compaction = {new_compaction_strategy_as_dict};".format(**locals())
+        alter_command_prefix = 'ALTER TABLE ' if keyspace_table not in mview_ks_cfs else 'ALTER MATERIALIZED VIEW '
+        cmd = alter_command_prefix + \
+            " {keyspace_table} WITH compaction = {new_compaction_strategy_as_dict};".format(**locals())
```

Comment:
we are in Python 3, use fstrings

Reply:
left the prior mode in order to be compatible with 2019.1 enterprise branch of python-2